static/app/views/eventsV2/chartFooter.tsx

```diff
@@ -45,6 +50,10 @@ export default function ChartFooter({
       <SectionValue key="total-value">{total.toLocaleString()}</SectionValue>
     )
   );
+  const topEventOptions: SelectValue<string>[] = [];
+  for (let i = 2; i <= 10; i++) {
```

Comment:
Not sure if there's an argument for allowing top-1?

Reply:
:+1: Good point, i'll update this to include 1 just so its more aesthetically pleasing